src/baseConfig.js

```diff
@@ -11,6 +11,8 @@ export class BaseConfig {
 
   constructor() {
     this._current = {
+      endpoint: null, // Endpoint to use for auth requests (local only). Null uses HttpClient.
```

Comment:
i assumed so. unfortunatelly i removed the line in the comment where i explicitly said said. it's just when i browsed the code i saw config.client and wondered what it was, so i checked here and didn't see it either. only later i spotted it in index.js. hence, the suggestion to add a comment here. like (old version):
client: null, // for internal use. the currently used rest client. gets set during configuration (see index.js)